src/Combinatorics/Matroids/quantum_automorphism_groups.jl

```diff
@@ -73,7 +73,14 @@ function _quantum_automorphism_group_indices(M::Matroid, structure::Symbol=:base
 
   rels = Vector{Tuple{Int,Int}}[]
 
-  sets  = getproperty(Oscar, structure)(M)
+  if structure == :bases
+    sets = bases(M)
+  elseif structure == :circuits
+    sets = circuits(M)
+  elseif structure == :flats
+    sets = flats(M)
+  end
```

Comment:
```suggestion
  else
    error("unreachable")
  end
```
this helps julia with inference, as otherwise it cannot eliminate the case of `sets` still being undefined